databricks/koalas/frame.py

```diff
@@ -7521,6 +7521,62 @@ def keys(self):
         """
         return self.columns
 
+    def pct_change(self, periods=1):
+        """
+        Percentage change between the current and a prior element.
+
```

Comment:
We should add a warning at `.. note::`

```
        .. note:: the current implementation of this API uses Spark's Window without
            specifying partition specification. This leads to move all data into
            single partition in single machine and could cause serious
            performance degradation. Avoid this method against very large dataset.
```

Reply:
@HyukjinKwon thanks for the comment! i'll add it !